dm/dm-ansible/roles/dm-master/defaults/main.yml

```diff
@@ -5,5 +5,7 @@
 dm_master_log_level: "info"
 dm_master_log_dir: "{{ deploy_dir }}/log"
 dm_master_log_file: "dm-master.log"
+dm_master_data_dir: "{{ deploy_dir }}/data.dm_master"
 dm_master_stdout_log_file: "dm-master-stdout.log"
 dm_master_stderr_log_file: "dm-master-stderr.log"
+dm_master_name_prefix: "dm_master"
```

Comment:
we often name the dm-master in `inventory.ini` as `dm_master`, after added this prefix, the name will become `dm_master_dm_master_8261`, is this a bit verbose?